### ISSUE TITLE
Server crashes on login using invalid data

### ISSUE THREAD

**Issue by aneesh-neelam** [aneesh-neelam/VITauth#2]
If no employee ID's are assigned for a given Semester/Slot/Room/Venue/Time on GetExamInfo, the MongoDB query returns a null result. But this crashes the server. 

Here are the logs: 

``` text
2015-04-28T17:13:23.618101+00:00 app[web.1]: ::ffff:10.13.14.157 - POST /api/client/GetExamInfo HTTP/1.1 200 41 - 2.894 ms
2015-04-28T17:13:32.937221+00:00 app[web.1]: TypeError: Cannot read property 'employee_ids' of null
2015-04-28T17:13:32.937224+00:00 app[web.1]:     at handleCallback (/app/node_modules/mongodb/lib/utils.js:95:12)
2015-04-28T17:13:32.937222+00:00 app[web.1]:     at onFindExam (/app/routes/api-client.js:95:33)
2015-04-28T17:13:32.937219+00:00 app[web.1]:                                         ^
2015-04-28T17:13:32.937226+00:00 app[web.1]:     at /app/node_modules/mongodb/lib/collection.js:849:5
2015-04-28T17:13:32.937227+00:00 app[web.1]:     at handleCallback (/app/node_modules/mongodb/node_modules/mongodb-core/lib/cursor.js:234:5)
2015-04-28T17:13:32.937229+00:00 app[web.1]:     at setCursorNotified (/app/node_modules/mongodb/node_modules/mongodb-core/lib/cursor.js:434:3)
2015-04-28T17:13:32.937230+00:00 app[web.1]:     at /app/node_modules/mongodb/node_modules/mongodb-core/lib/cursor.js:537:16
2015-04-28T17:13:32.937232+00:00 app[web.1]:     at queryCallback (/app/node_modules/mongodb/node_modules/mongodb-core/lib/cursor.js:215:5)
2015-04-28T17:13:32.937236+00:00 app[web.1]:     at null.messageHandler (/app/node_modules/mongodb/node_modules/mongodb-core/lib/topologies/server.js:218:23)
2015-04-28T17:13:32.937234+00:00 app[web.1]:     at Callbacks.emit (/app/node_modules/mongodb/node_modules/mongodb-core/lib/topologies/server.js:84:3)
2015-04-28T17:13:32.937238+00:00 app[web.1]:     at Socket.<anonymous> (/app/node_modules/mongodb/node_modules/mongodb-core/lib/connection/connection.js:259:22)
2015-04-28T17:13:32.937217+00:00 app[web.1]:     process.nextTick(function() { throw err; });
2015-04-28T17:13:32.937174+00:00 app[web.1]: /app/node_modules/mongodb/lib/utils.js:97
```

**Comment by kishore-narendran** [comment]
I'll fix it. 

-----Original Message-----

**Comment by kishore-narendran** [comment]
Fixed in 5a1420d93dd47d6742932435c8bcc8142909df68
